content/tutorials/examples/phel-in-a-nutshell.md

```diff
@@ -0,0 +1,38 @@
++++
+sort_by = "weight"
++++
+
+# Phel in a nutshell
+
+```phel
+(defn- fill-keys
+  "Exchanges all keys with their associated values in an array."
+  [xs v]
+  (-> (to-php-array xs)
+      (php/array_fill_keys v)
+      (php-array-to-map)))
```

Comment:
What do you think about this function, @jenshaase? Should we have something like this in the core? 
I think it's generic enough and it doesn't "belong" only to the logic of this example...

Reply:
Don't see a good reason for that. In case of the example code here, it even makes more sense to use a set instead of an array.

Reply:
I don't understand what do you mean by "use a set instead of an array"? Can you please write down the code that you have in your mind with that?

Reply:
* `(def top-english-words (set "the" "he" "at" ...))`
* remove `words-to-filter-out` (not needed)
* Instead of `(filter |(not (get words-to-filter-out $))` do `(filter |(nil? (top-words $)) words)`

Reply:
Got it! Totally right. I will apply this change